cmd/vic-machine/create/create.go

```diff
@@ -1610,7 +1610,18 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	// If the supplied vSphere network contains spaces then the user must supply a network alias. Guest info won't receive a name with spaces.
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
In other words, they forgot to specify an alias and just added a colon the the vsphere network.